.github/workflows/enos-run.yml

```diff
@@ -29,6 +29,7 @@ jobs:
           - filter: 'e2e_database'
           - filter: 'e2e_static builder:crt'
           - filter: 'e2e_static_with_vault builder:crt'
+          # - filter: 'e2e_ui builder:crt' # Don't run UI tests yet. takes too long.
```

Comment:
We're planning on running these tests locally for now until we can work out some speed gains.